api/RTCPeerConnection.json

```diff
@@ -1639,89 +1523,69 @@
               "version_added": false
             },
             "opera": {
-              "version_added": "45"
+              "version_added": "15"
             },
             "opera_android": {
-              "version_added": "43"
+              "version_added": "14"
             },
             "safari": {
               "version_added": "11"
             },
             "safari_ios": {
               "version_added": "11"
             },
-            "samsunginternet_android": [
-              {
-                "version_added": "7.0",
-                "notes": "Promise resolves with <code>RTCStatsReport</code>."
-              },
-              {
-                "version_added": "6.0",
-                "notes": "Promise-based version."
-              },
-              {
-                "version_added": true
-              }
-            ],
-            "webview_android": [
-              {
-                "version_added": "58",
-                "notes": "Promise resolves with <code>RTCStatsReport</code>."
-              },
-              {
-                "version_added": "54",
-                "notes": "Promise-based version."
-              },
-              {
-                "version_added": true
-              }
-            ]
+            "samsunginternet_android": {
+              "version_added": "1.5"
+            },
+            "webview_android": {
+              "version_added": "≤37"
+            }
           },
           "status": {
             "experimental": false,
             "standard_track": true,
             "deprecated": false
           }
         },
-        "selector_parameter": {
+        "returns_promise": {
           "__compat": {
-            "description": "<code>selector</code> parameter",
+            "description": "Returns a <code>Promise</code>",
             "support": {
               "chrome": {
-                "version_added": "67"
+                "version_added": "51"
```

Comment:
Was both 54 and 58 in the original data wrong?

Reply:
Looks like there were some problems with this data, fixed!